examples/hockey/hockey_game.py

```diff
@@ -78,14 +89,14 @@ def _update_environment(self):
             self.winner = self.info["winner"]
             if self.winner == 1:
                 if self.sides_swapped:
-                    self.score[1] = self.score[1] + 1
+                    self.scores[self.player_2_id] = self.scores[self.player_2_id] + 1
```

Comment:
This is fine, but `+= 1` would be better.

Reply:
Ahh yes, didn't think of that